### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by Jason-Cooke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ or failure.
 Available options:
 
 + `-n`: Suppresses the normal change of directory when adding directories to the stack, so that only the stack is manipulated.
-+ `-q`: Supresses output to the console.
++ `-q`: Suppresses output to the console.
```
Comment (nfischer) [inline README.md]: Could you please make this change here instead:

https://github.com/shelljs/shelljs/blob/aa9d443e7b8126763cfb76b1230a7bdb578a6a5e/src/dirs.js#L43

re-generate the README with `npm run gendocs`

Comment (Jason-Cooke) [reply README.md]: Ya, consider it done :)

**Comment by nfischer** [comment]
Thanks! Not sure why codecov is confused about code coverage.

**Comment by Jason-Cooke** [comment]
No problem!

**Pull request merged**